prov/usnic/src/fi_ext_usnic.h

```diff
@@ -42,7 +42,7 @@
 #include <stdint.h>
 #include <net/if.h>
 
-#define FI_PROTO_RUDP 100
+#define FI_PROTO_RUDP (100 | FI_PROV_SPECIFIC)
```

Comment:
Yeah.  The `protocol_version` field is a `uint32_t`, while `FI_PROV_SPECIFIC` is defined as `(1 << 31)`, which probably works out fine when assigned to a `uint32_t`, but requires careful thought about the C standard instead of just forcing everything to be unsigned and being trivially certain.  Actually, now that I think about it, `FI_PROV_SPECIFIC` should probably also be defined to `(1u << 31)` instead.